etc/bash_completion.d/yab

```diff
@@ -0,0 +1,16 @@
+# bash completion for yab executable
+have yab &&
```

Comment:
what is `have yab`?

Reply:
The meaning is: is there a `yab` executable somewhere in `$PATH`. This prevents the autocomplete functionality from loading if there is no yab executable available. Too many autocomplete functions make the shell run slower. This also allows for splitting the shipping of the yab binary and its autocomplete functionality.
`have` is documented [here](https://salsa.debian.org/debian/bash-completion/blob/master/bash_completion#L113). While it is deprecated in newer versions, brew [uses older versions](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bash-completion.rb#L6) so for now I am stuck with it.

Reply:
Useful stack question about this
https://stackoverflow.com/questions/12874920/have-keyword-for-bash-completion/12874971#12874971

Since this is a standard check for bash, seems fine.